sonar-ws/src/main/java/org/sonarqube/ws/client/users/UsersService.java

```diff
@@ -74,7 +74,7 @@ public CreateWsResponse create(CreateRequest request) {
         .setParam("login", request.getLogin())
         .setParam("name", request.getName())
         .setParam("password", request.getPassword())
-        .setParam("scmAccount", request.getScmAccount())
+        .setParam("scmAccount", request.getScmAccount() == null ? null : request.getScmAccount())
         .setParam("scmAccounts", request.getScmAccounts() == null ? null : request.getScmAccounts().stream().collect(Collectors.joining(","))),
```

Comment:
I see other tests remove `scmAccounts` param in favor of singular `scmAccount`, used with list values.

Also, I don't see how this change is related to the task at hand with pull requests.

Reply:
I did this while updating the WS client, the scmAccounts parameter is deprecated